### PR TITLE
added 'current_enable_status' endpoint details

### PR DESCRIPTION
**Pull request by Wajahat-Bboxx**

### CODE REVIEW

File: source/includes/_current_enable_status.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+## `/v1/current_enable_status`
+
+> A GET request to this endpoint will return the enable status of all units so that the ERP can sync the unit status
+
+```python
+import requests
+
+url = "https://smartapi.bboxx.co.uk/v1/current_enable_status?page=5&results_per_page=3"
+
+payload={}
```
Comment (mujeeb-bboxx) [inline source/includes/_current_enable_status.md]: This is a GET request, so there is no need to add `payload`.

File: source/includes/_current_enable_status.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+headers = {
+  'Authorization': 'Token token='+A_VALID_TOKEN
+}
```
Comment (mujeeb-bboxx) [inline source/includes/_current_enable_status.md]: Use same header as in other files, which is this,
`headers = {'Content-Type': 'application/json', 'Authorization': 'Token token=A_VALID_TOKEN'}`

File: source/includes/_current_enable_status.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+response = requests.request("GET", url, headers=headers, data=payload)
```
Comment (mujeeb-bboxx) [inline source/includes/_current_enable_status.md]: No need to add `payload` and use `requests.get(url=url, headers=headers)` instead.

File: source/index.html.md [hunk 4]
```diff
@@ -156,6 +156,7 @@ includes:
   - custom_endpoints
   - home
   - auth_login
+  - current_enable_status.md
```
Comment (mujeeb-bboxx) [inline source/index.html.md]: No need to add _.md_

**Review by mujeeb-bboxx — CHANGES_REQUESTED** [review]

File: source/includes/_current_enable_status.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+```python
+import requests
+
+url = "https://smartapi.bboxx.co.uk/v1/current_enable_status?page=5&results_per_page=3"
+
+
+headers = {'Content-Type': 'application/json', 'Authorization': 'Token token=A_VALID_TOKEN'}
+
+response = requests.request("GET", url, headers=headers)
```
Comment (mujeeb-bboxx) [inline source/includes/_current_enable_status.md]: Use `requests.get(url=url, headers=headers)` instead.

**Review by mujeeb-bboxx — CHANGES_REQUESTED** [review]

File: source/includes/_current_enable_status.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
+## `/v1/current_enable_status`
+
+> A GET request to this endpoint will return the enable status of all units so that the ERP can sync the unit status
+
+```python
+import requests
+
```
Comment (mujeeb-bboxx) [inline source/includes/_current_enable_status.md]: Please remove these two lines.

**Review by mujeeb-bboxx — CHANGES_REQUESTED** [review]

**Review by barkatkhan-bboxx — APPROVED** [review]

**Review by mujeeb-bboxx — APPROVED** [review]

**Pull request merged**